include/nihstro/shader_bytecode.h

```diff
@@ -379,47 +379,68 @@ struct OpCode {
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments, "mul" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments, "sge" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments, "slt" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::OneArgument, "flr" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments, "max" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments, "min" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::OneArgument, "rcp" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::OneArgument, "rsq" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::MOVA, "mova" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::OneArgument, "mov" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments | OpCode::Info::SrcInversed, "dphi" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments | OpCode::Info::SrcInversed, "sgei" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::TwoArguments | OpCode::Info::SrcInversed, "slti" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Trivial, 0, "nop" }
+            , { OpCode::Type::Trivial, 0, "end" }
+            , { OpCode::Type::Conditional, OpCode::Info::BREAKC, "breakc" }
+            , { OpCode::Type::Conditional, OpCode::Info::CALL, "call" }
+            , { OpCode::Type::Conditional, OpCode::Info::CALLC, "callc" }
+            , { OpCode::Type::UniformFlowControl, OpCode::Info::CALLU, "callu" }
+            , { OpCode::Type::UniformFlowControl, OpCode::Info::IFU, "ifu" }
+            , { OpCode::Type::Conditional, OpCode::Info::IFC, "ifc" }
+            , { OpCode::Type::UniformFlowControl, OpCode::Info::LOOP, "loop" }
+            , { OpCode::Type::Trivial, 0, "emit" }
+            , { OpCode::Type::SetEmit, 0, "setemit" }
+            , { OpCode::Type::Conditional, OpCode::Info::JMPC, "jmpc" }
+            , { OpCode::Type::Conditional, OpCode::Info::JMPU, "jmpu" }
+            , { OpCode::Type::Arithmetic, OpCode::Info::Compare, "cmp" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::MultiplyAdd, OpCode::Info::SrcInversed, "madi" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
+            , { OpCode::Type::Unknown, 0, "UNK" }
```

Comment:
Please use an `static OpCode::Info dummy` object for elements which are not actually unknown but instead will never be looked up anyway.